OpenRA.Mods.Cnc/Util.cs

```diff
@@ -17,26 +17,54 @@ public static class Util
 		// This table defines the exclusive maximum facing for the i'th sprite frame.
 		// i.e. sprite frame 1 is used for facings 5-13, sprite frame 2 for 14-21, and so on.
 		// Sprite frame 0 is used for facings smaller than 5 or larger than 249.
-		static readonly int[] SpriteFacings =
+		static readonly int[] SpriteRanges =
 		{
-			5, 14, 22, 33, 39, 46, 53, 60,
-			67, 74, 81, 88, 96, 104, 113, 122,
-			133, 142, 151, 161, 167, 174, 181, 188,
-			195, 202, 209, 216, 224, 232, 241, 250
+			20, 56, 88, 132, 156, 184, 212, 240,
+			268, 296, 324, 352, 384, 416, 452, 488,
+			532, 568, 604, 644, 668, 696, 724, 752,
+			780, 808, 836, 864, 896, 928, 964, 1000
 		};
 
-		public static int ClassicQuantizeFacing(int facing, int numFrames, bool useClassicFacingFudge)
+		// The actual facing associated with each sprite frame,
+		// calculated as the mid-point of the ranges defined above
+		static readonly WAngle[] SpriteFacings =
 		{
-			if (useClassicFacingFudge && numFrames == 32)
+			WAngle.Zero, new WAngle(40), new WAngle(74), new WAngle(112), new WAngle(146), new WAngle(172), new WAngle(200), new WAngle(228),
```

Comment:
As discussed on IRC: We'll need to recheck how this was calculated.

Reply:
These were generated using
```python
ranges = [
	5, 14, 22, 33, 39, 46, 53, 60,
	67, 74, 81, 88, 96, 104, 113, 122,
	133, 142, 151, 161, 167, 174, 181, 188,
	195, 202, 209, 216, 224, 232, 241, 250
]

print('ranges = {' + ', '.join([str(4*i) for i in ranges]) + '}')
facings = [0]
for i in range(1, len(ranges)):
    facings.append(4*(ranges[i] + ranges[i-1] + 1) // 2)

print('steps = {' + ', '.join(['new WAngle({})'.format(i) for i in facings]) + '}')
```

Reply:
I didn't really spot any difference ingame when I "corrected" the values here by applying the mid-point rule to the current ranges (that are 4 times larger). So I guess we can keep that, but maybe should update the comment.